### ISSUE TITLE
Simplify `scda` calls

### ISSUE THREAD

**Issue by danielinteractive** [insightsengineering/biomarker-catalog#50]
We should use calls such as
`adsl <- synthetic_cdisc_dataset("latest", "adsl")` 
consistently to avoid too specific calls such as
`scda::synthetic_cdisc_data("rcd_2022_06_27")$adsl`
because it complicates installation, e.g. after installing `scda` you still get the error `"No synthetic CDISC data archive packages are installed"`

**Comment by shajoezhu** [comment]
yes, @danielinteractive , we can close this one with https://github.com/insightsengineering/biomarker-catalog/pull/64
